### ISSUE TITLE
gotk3 is very good,but it warning every time

### ISSUE THREAD

**Issue by biij5698** [gotk3/gotk3#211]
WARNING **: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files

**Comment by biij5698** [comment]
I love gotk3 and use it every day

**Comment by founderio** [comment]
This warning is likely caused by your system setup, not related to gotk3. Trylaunching a different gtk application from a terminal and check if the message appears there as well.

It looks like your system does not have an accessibility service available - in itself this is not a problem, I would assume.  
Maybe this thread can help you disable that message if it really bothers you: https://forums.gentoo.org/viewtopic-t-947210-start-0.html
